flake8_trio/visitors/visitor91x.py

```diff
@@ -54,148 +106,232 @@ def __init__(self, *args: Any, **kwargs: Any):
         self.has_yield = False
         self.safe_decorator = False
         self.async_function = False
-
         self.uncheckpointed_statements: set[Statement] = set()
-        self.uncheckpointed_before_continue: set[Statement] = set()
-        self.uncheckpointed_before_break: set[Statement] = set()
 
-        self.default = self.get_state()
+        self.loop_state = LoopState()
+        self.try_state = TryState()
+
+    @property
```

Comment:
I elected to add a million property and setters instead of rewriting everything refer to `self.loop_state.xxx`. In large part to make the diff smaller / easier to read, but I don't care much for them otherwise.